### ISSUE TITLE
MacOS failures

### ISSUE THREAD

**Issue by jngrad** [espressomd/espresso#2829]
MacOS jobs are failing:
https://gitlab.icp.uni-stuttgart.de/espressomd/espresso/pipelines/7224
https://gitlab.icp.uni-stuttgart.de/espressomd/espresso/pipelines/7225
```
fatal: unable to access 'https://gitlab-ci-token:[MASKED]@gitlab.icp.uni-stuttgart.de/espressomd/espresso.git/':
SSL certificate problem: certificate is not yet valid
ERROR: Job failed: Process exited with: 1. Reason was:  ()
```

**Comment by KaiSzuttor** [comment]
we recently had issues with our webserver certificate. those should be resolved by now

**Comment by jngrad** [comment]
still failing

**Comment by mkuron** [comment]
Fixed now. This was caused by the certificate issue, but is just another variant of the problem where virtual machines don't have the correct time until a few seconds/minutes after they are started. I have now resolved the latter by disabling snapshots so that a fresh VM (with correct time) is booted for every build. This adds about a minute to the build time, but should also fix the occasional timeouts where tests claimed to have timed out after millions of seconds.
